java/flight/flight-core/pom.xml

```diff
@@ -86,6 +86,10 @@
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java-util</artifactId>
```

Comment:
I assumed it would take from the bom? The same version used for `protobuf-java`, I could be wrong.

Reply:
We don't use a bom for protobuf-java.

Reply:
Ah I see. I will specify it here explicitly here.

Reply:
Adding `<version>3.23.1</version>`

Reply:
No, can we use the `protobuf.version` property to be consistent?

Reply:
gah, sorry, I realized we use a mix of BOM and protobuf.version...let's revert this and go back to using the BOM.

Can you file an issue to have Gandiva and Dataset use the BOM instead of defining their own protobuf.version?

Reply:
@lidavidm solved the issue here and filed an issue to fix Gandiva and Dataset: https://github.com/apache/arrow/issues/39727